### DISCUSSION TITLE
Iphone help

### DISCUSSION THREAD

**Discussion by nagydavid** [ESPresense/ESPresense#44] [Q&A]
Hi, 

I would like to get some help regarding iphone. I have two iphone  an Iphone X and 11. When the X is locked the BLE packets are kept transmitting, however the 11 is only transmitting when it is unlocked. I tried to map out all the possible setting, but nothing is keeping the 11 transmitting. May be (probably), I am not doing something right, but I would like to here your opinions. 

Thanks 
D. 

Edit.: The X is a company phone with restrictions, so it makes it even weirder

**Comment by DTTerastar** [comment]
This is a great article about what's transmitted and when: https://petsymposium.org/2020/files/papers/issue1/popets-2020-0003.pdf

**Comment by DTTerastar** [comment]
>An underlying condition for Continuity messages to besent is the user having their device associated with aniCloud account to which at least two devices are reg-istered; because users regularly neglect to remove oldApple products from their iCloud account years afterdiscarding or retiring the device, this condition is rou-tinely met even when the user may only actively use oneApple device.While they are enabled by default, Handoff mes-sages described in Section 4.1.6 are unique in that theycan be explicitly turned off in the Settings Menu. Somemessage types require a device with a cellular connec-tion to be associated with the iCloud account in orderto be generated by the user, namely the WiFi Settingsand Instant Hotspot messages, outlined respectively inSections 4.1.7 and 4.1.8, which need a cellular-capabledevice to act as a hotspot.Activating Airplane Mode does not disable thetransmission of Continuity messages in either iOS 11or 12, regardless of whether Airplane Mode is activatedfrom the Control Center or Settings Menu. Similarly,disabling Bluetooth from the Control Center in iOS 11or 12 does not discontinue the transmission of Continu-ity messages [3]. We determined that the only way tostop transmission of all Continuity messages is to dis-able Bluetooth from the Settings Menu.
